lib/plugins/aws/provider/awsProvider.js

```diff
@@ -454,6 +454,20 @@ class AwsProvider {
     return regionSourceValue.value || defaultRegion;
   }
 
+  getRuntimeSourceValue() {
+    const values = this.getValues(this, [
+      ['options', 'runtime'],
+      ['serverless', 'config', 'runtime'],
```

Comment:
Same, we do not support it as top level config option, and I don't think it should change with this PR

Reply:
ok